ruff.toml

```diff
@@ -0,0 +1,6 @@
+[lint]
+ignore = [
+  "E402",  # Module level import not at top of file -> gi imports may come after gi.require_version.
+  "E501",  # Line too long. This is probably better enforced manually where it makes sense.
+]
+select = [ "W", "E", "F", "ARG"]
```

Comment:
What do you mean by two files?
I will move it to langs/python for now, but this makes running it locally a bit more complicated, though it's not a big issue since I can just configure it in the Makefile.

Reply:
Can we keep it one file for now? We can split it up later if needed anyway.

Workbench only has a handful of Python files internally, I doubt we couldn't use the same rule. I want to keep the rules as simple as possible in any case.